frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -317,13 +309,25 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
               position={[latitude, longitude]}
               icon={getMarkerIcon(cluster)}
               onclick={() => {
+                const convertedProperty = convertToProperty(
+                  cluster.properties,
+                  latitude,
+                  longitude,
+                );
+                //sets this pin as currently selected
+                if (
+                  cluster.properties.propertyTypeId === PropertyTypes.PARCEL ||
+                  cluster.properties.propertyTypeId === PropertyTypes.SUBDIVISION
+                ) {
+                  dispatch(parcelsActions.storeParcelDetail(convertedProperty as IParcel));
```

Comment:
Correct me if I'm wrong that a Subdivision is still a Parcel

Reply:
this is correct.